src/odb/include/odb/db.h

```diff
@@ -8895,6 +8899,38 @@ class dbTechLayerEolKeepOutRule : public dbObject
   // User Code End dbTechLayerEolKeepOutRule
 };
 
+class dbTechLayerForbiddenSpacingRule : public dbObject
```

Comment:
There is no way to tell if the optional fields have been given in the LEF or not.  Usually we have has<field>() methods (see dbTechLayerSpacingRule for an example)

Reply:
I've added in the following way:
<img width="628" alt="image" src="https://github.com/The-OpenROAD-Project/OpenROAD/assets/43450810/85ef3a2e-30e9-43b5-a8f0-853a5c6ddd2a">

I'm comparing against the default value of those fields. Is this the correct way to do it?